app/components/templates/Main/Main.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+import Nav from '../../organisms/Nav/Nav';
+import Link from '../../../components/atoms/Link/Link.js'
+
+export default class Main extends React.Component {
+  constructor(props) {
+      super(props);
+  }
+
+  render() {
+    return(
+      <div className="app">
+        <Nav />
+        
+        <main>
```

Comment:
We don't, I just figured an identifier tag for the output would be better than just calling the component.